csrankings-1.csv

```diff
@@ -562,7 +562,7 @@ Caetano Traina,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/caetano,h2ivd3AAAAAJ
 Caetano Traina Jr.,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/caetano,h2ivd3AAAAAJ
 Caetano Traina Junior,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/caetano,h2ivd3AAAAAJ
 Cagatay Goncu,Monash University,http://monash.edu/research/explore/en/persons/cagatay-goncu(8747e864-156a-441d-9287-1bc0adff8611).html,AjUOkTcAAAAJ
-Cagatay Turkay,City University of London,https://www.city.ac.uk/people/academics/cagatay-turkay#profile=overview,ho0I_1kAAAAJ
+Cagatay Turkay,University of Warwick,https://warwick.ac.uk/fac/cross_fac/cim/people/cagatay-turkay/,ho0I_1kAAAAJ

```

Comment:
Not clearly eligible for inclusion. Please point to a CS department affiliation or similar.

Reply:
Doesn't seem to be pure CS department as you figured out; should I remove him?